third_party/2and3/itsdangerous.pyi

```diff
@@ -105,8 +105,8 @@ class Serializer(object):
     def iter_unsigners(self, salt: Optional[Union[Text, bytes]] = ...) -> Generator[Any, None, None]: ...
     def dumps(self, obj: Any, salt: Optional[Union[Text, bytes]] = ...) -> Any: ...  # morally -> Union[str, bytes]
     def dump(self, obj: Any, f: IO[Any], salt: Optional[Union[Text, bytes]] = ...) -> None: ...
-    def loads(self, s: Union[Text, bytes], salt: Optional[Union[Text, bytes]] = ...) -> Any: ...
-    def load(self, f: IO[Any], salt: Optional[Union[Text, bytes]] = ...): ...
+    def loads(self, s: Union[Text, bytes], salt: Optional[Union[Text, bytes]] = ...) -> Optional[Exception]: ...
+    def load(self, f: IO[Any], salt: Optional[Union[Text, bytes]] = ...) -> Optional[Exception]: ...
```

Comment:
That doesn't look right to me. Looking at the [source code](https://github.com/pallets/itsdangerous/blob/5574351e275cbe742a7e0c5a8cb3fec9a3c09d89/src/itsdangerous/serializer.py#L178-L193), `loads()` returns the result of `last_payload()`. The latter is annotated as returning `Any`, but according to the docstring returns `bytes`.

Reply:
Good catch. I misread the source the first time.